tests/backup/backup_helper.go

```diff
@@ -291,7 +291,8 @@ func CreateBackupWithCustomResourceType(backupName string, clusterName string, b
 func CreateScheduleBackup(backupName string, clusterName string, bLocation string, bLocationUID string,
 	namespaces []string, labelSelectors map[string]string, orgID string, preRuleName string,
 	preRuleUid string, postRuleName string, postRuleUid string, schPolicyName string, schPolicyUID string, ctx context.Context) (string, error) {
-	var bkpUid string
+	var schBkpName string
+	var schBkpUid string
```

Comment:
Change this to `scheduledBackupName` and `scheduledBackupUid`

Reply:
I am considering changing it to recentScheduleBackupName and recentScheduleBackupUid